app/assets/stylesheets/components/task_list.scss

```diff
@@ -0,0 +1,65 @@
+// Task list pattern
```

Comment:
I think it would be good to reference the place this SCSS came from as we haven't written it ourselves. That way we give a hint to future developer that it might need updating or may become redundant if/when it stops being an experimental pattern and becomes actually part of `govuk-frontend`. We should be able to reference it in the GOV.UK Prototype Kit.

Reply:
Yea good idea 👍